packages/semantic-tokens/scripts/fluentOverrides.ts

```diff
@@ -8,21 +8,122 @@ export type FluentOverrideValue =
   ctrlLinkForegroundBrandPressed: { f2Token: 'colorBrandForegroundLinkPressed' },
   ctrlLinkForegroundNeutralRest: { f2Token: 'colorNeutralForeground2' },
   ctrlLinkForegroundNeutralPressed: { f2Token: 'colorNeutralForeground2Pressed' },
   ctrlLinkForegroundNeutralHover: { f2Token: 'colorNeutralForeground2Hover' },
   foregroundCtrlNeutralPrimaryDisabled: { f2Token: 'colorNeutralForegroundDisabled' },
+  backgroundCtrlNeutralRest: { f2Token: 'colorNeutralBackground1' },
+  foregroundCtrlNeutralPrimaryRest: { f2Token: 'colorNeutralForeground1' },
+  strokeCtrlOnneutralRest: { f2Token: 'colorNeutralStroke1' },
+  backgroundCtrlNeutralHover: { f2Token: 'colorNeutralBackground1Hover' },
+  strokeCtrlOnneutralHover: { f2Token: 'colorNeutralStroke1Hover' },
+  foregroundCtrlNeutralPrimaryHover: { f2Token: 'colorNeutralForeground1Hover' },
+  backgroundCtrlNeutralPressed: { f2Token: 'colorNeutralBackground1Pressed' },
+  strokeCtrlOnneutralPressed: { f2Token: 'colorNeutralStroke1Pressed' },
+  foregroundCtrlNeutralPrimaryPressed: { f2Token: 'colorNeutralForeground1Pressed' },
+  textRampItembodyFontsize: { f2Token: 'fontSizeBase300' },
+  textCtrlButtonWeightDefault: { f2Token: 'fontWeightSemibold' },
+  textRampItembodyLineheight: { f2Token: 'lineHeightBase300' },
+  ctrlFocusInnerStroke: { f2Token: 'colorStrokeFocus2' },
+  ctrlFocusOuterStrokewidth: { f2Token: 'strokeWidthThick' },
+  cornerCtrlRest: { f2Token: 'borderRadiusMedium' },
+  strokeLayer: { f2Token: 'colorTransparentStroke' },
+  backgroundCtrlBrandRest: { f2Token: 'colorBrandBackground' },
+  backgroundCtrlBrandHover: { f2Token: 'colorBrandBackgroundHover' },
+  foregroundCtrlOnbrandRest: { f2Token: 'colorNeutralForegroundOnBrand' },
+  foregroundCtrlOnbrandHover: { f2Token: 'colorNeutralForegroundOnBrand' },
+  foregroundCtrlOnbrandPressed: { f2Token: 'colorNeutralForegroundOnBrand' },
+  backgroundCtrlSubtleRest: { f2Token: 'colorSubtleBackground' },
+  foregroundCtrlOnsubtleRest: { f2Token: 'colorNeutralForeground2' },
+  backgroundCtrlSubtleHover: { f2Token: 'colorSubtleBackgroundHover' },
+  backgroundCtrlSubtlePressed: { f2Token: 'colorSubtleBackgroundPressed' },
+  foregroundCtrlOnsubtleHover: { f2Token: 'colorNeutralForeground2Hover' },
+  foregroundCtrlOnsubtlePressed: { f2Token: 'colorNeutralForeground2Pressed' },
+  foregroundCtrlIconOnsubtlePressed: { f2Token: 'colorNeutralForeground2BrandPressed' },
+  foregroundCtrlIconOnsubtleHover: { f2Token: 'colorNeutralForeground2BrandHover' },
+  strokeCtrlOnactivebrandRest: { f2Token: 'colorNeutralForegroundOnBrand' },
+  strokeCtrlOnactivebrandHover: { f2Token: 'colorStrokeFocus2' },
+  cornerCircular: { f2Token: 'borderRadiusCircular' },
+  paddingCtrlLgHorizontalDefault: { f2Token: 'spacingHorizontalL' },
+  paddingCtrlSmHorizontalDefault: { f2Token: 'spacingHorizontalS' },
+  cornerCtrlSmRest: { f2Token: 'borderRadiusMedium' }, // Button uses borderRadiusMedium in all sizes
```

Comment:
We will move this to inline style fallbacks if it conflicts with other components

Reply:
Is the idea to catch this with visual diff?

Reply:
No, this VR diff is correct for button, but other components may not follow this fallback.

We'll need to see how other components implement corner radius (and whether other components use the small/medium/large generics) to see if this needs to be inline fallbacks via styles file (i.e. generic token falls back to different tokens in different components).

Since most components should use the optional level tokens we can have variant fallbacks regardless so it's likely not an issue.